e2e/paths/enterAllocationProposal_test.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../config.js');
+
+Feature('EnterAllocationProposal');
```

Comment:
Let's use plain English here.

Reply:
How codecptjs configures the file when you generate test files from the command line, shall fix